helpdesk_mgmt_project/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Helpdesk Project',
+    'summary': """
+        Helpdesk""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'After-Sales',
+    'author': 'PuntSistemes S.L.U., '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/helpdesk',
+    'depends': [
+        'helpdesk_mgmt',
+        'project',
+    ],
+    'data': [
+        'views/helpdesk_ticket_view.xml',
+        'views/project_view.xml',
+        'views/project_task_view.xml',
+    ],
+    'demo': [
+    ],
+    'development_status': 'Mature',
```

Comment:
This can't be mature not having tests nor being in several versions:

https://odoo-community.org/page/module-maturity-levels

Reply:
@cramosh , do you want me to take care of the changes and tests to unlock the PR?

Reply:
Mature is not possible any way. Production/Stable is.

Reply:
Yes, you can made the changes. I need your help for design test.

Reply:
> Mature is not possible any way. Production/Stable is.

I know,I will add tests and change to Production status.  Thanks!

> Yes, you can made the changes. I need your help for design test.

Then I will fork your branch and create new PR with the changes.